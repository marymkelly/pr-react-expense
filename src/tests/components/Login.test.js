import React from 'react';
import { shallow } from 'enzyme';
import { Login } from '../../components/pages/Login.js';

test('should render Login corectly', () => {
    const wrapper = shallow(<Login />)
    expect(wrapper).toMatchSnapshot();
})

test('should call startLogin', () => {
    const startLogin = jest.fn();
    const wrapper = shallow(<Login startLogin={startLogin} />)
    wrapper.find('button').simulate('click');
    expect(startLogin).toHaveBeenCalled();
})
