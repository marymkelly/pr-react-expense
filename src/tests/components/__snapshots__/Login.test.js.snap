// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Login corectly 1`] = `
<div>
  <button>
    Log In
  </button>
</div>
`;
